stock-service/src/main/java/rs/banka4/stock_service/runners/TestDataRunner.java

```
@@ -363,13 +360,19 @@ private Stock fetchStockInfo(String ticker, long id) throws IOException {
             return null;
         }
 
+        BigDecimal marketCap = null;
+        try {
+            marketCap = new BigDecimal(stockInfoDto.marketCap());
+        } catch (Exception e) {
```

don't ignore this exception - let the program crash if it happens (presumably, it should not be possible, right?)

---

This is a @transient variable we can calculate once we get a price. The API often has this field as None. For example Apple would crash / skip if I didnt just set it to null.